canonical/ubuntu.com
# Upgrade vanilla and global nav versions

## Done

- Upgrade Vanilla version to 4.9.1
- Upgrade global nav to 3.6.4

## QA

- View the site locally in your web browser at: https://ubuntu-com-13718.demos.haus/
- Check as many pages as possible and see that nothing looks broken

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
